app/src/main/java/org/openobservatory/ooniprobe/activity/ResultDetailActivity.java

```diff
@@ -105,6 +110,54 @@ protected void onResume() {
         load();
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.rerun, menu);
```

Comment:
Why do you need to inflate the menu explicitly here?

Reply:
To show the reload icon on the top right

Reply:
Ah, so `.inflate(R.menu.rerun, menu)` is adding the menu entry, I see, thanks!